webviews/welcome/index.html

```diff
@@ -0,0 +1,15 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title></title>
+    <style-inject></style-inject>
+</head>
+<body>
+<h1>Welcome to Zowe Explorer!</h1>
+<h2>You see an example of intro screen which supports <span class="css-style">CSS</span> and <span
```

Comment:
would this wording be merged as-is - or will other messaging appear here?

Reply:
No, it won't be, once I receive the production ready text and styles it will be updated. This PR presents an approach and opens it for the discussion. Content of the view will be updated later.

Reply:
ok i think i got it (finally) 😄